### DISCUSSION TITLE
Expand Crossing Marking Quest

### DISCUSSION THREAD

**Discussion by RubenKelevra** [streetcomplete/StreetComplete#5875] [Ideas]
In SCEE, there’s currently a quest for mapping crossing markings. To prevent overwhelming users with irrelevant options, it would be useful to create a list of official crossing markings for each country.

If you know the official crossing markings in your country, please share them below! Once we’ve collected enough data from different countries, it can be incorporated into StreetComplete to make the selection process more streamlined and accurate.

Looking forward to your contributions!

**Comment by RubenKelevra** [comment]
For Germany, the following values are in use:

- `yes`
- `no`
- `zebra`
- `dashes`

Plus some rarely used values or errors.

![Screenshot_20240905_142930](https://github.com/user-attachments/assets/6d4a15da-3dcc-4e98-aec3-d69c0118165d)

But at least around here I've got a lot of dashes with pictograms of bikes, which I'm not sure which value to use there.

![Screenshot_20240905_143425](https://github.com/user-attachments/assets/69ca9eb9-462f-412b-a2fd-2ce0aa7b1542)

Plus we got such combines zebra crossing, with bicycle section which is dashed, with pictograms and with solid color.

So `crossing:markings=zebra;surface+dashes+pictograms`?

![640px-Kombinierte_Querungsanlage_Radweg_Zebrastreifen](https://github.com/user-attachments/assets/630ac184-51a3-4275-b362-eb12d93dfcd7)

**Comment by mnalis** [comment]
- (ref back to issue: https://github.com/streetcomplete/StreetComplete/issues/5872#issuecomment-2331767051)

As for [Croatia](https://taginfo.geofabrik.de/europe:croatia/keys/crossing:markings#values), AFAICT it is mostly `zebra` (usually marked just as `yes` without details) or `no` if we are talking about pedestrian crossings; or for `bicycle=designated` + `foot=designated` + `segregated=yes` then (according to the [wiki](https://wiki.openstreetmap.org/wiki/Key:crossing:markings#More_difficult_cases)) usually one of:

- `footway:crossing:markings=zebra` + `cycleway:crossing:markings=dots;surface`, _or_
- `footway:crossing:markings=zebra` + `cycleway:crossing:markings=dots`, _or_
- `footway:crossing:markings=zebra` + `cycleway:crossing:markings=surface`

with very little other values.... But as they are all legally the same regardless of small variations in markings (i.e. determined whether they have `bicycle=designated` or `bicycle=no`), I must say I don't see much usefulness in expending effort tagging differences between them.

**Reply by RubenKelevra** [reply]
Well in case of Germany the differences result in you're legally able to ride your bike over them, or you have to get off it and walk across, which makes a difference in travel time for routing - and obviously convenience ;)

Same for walking, we got zebra crossings, where you have priority as pedestrian, or regular crossings, where you may have to wait for a minute or so to cross.

So a router may choose the crossing which is most convenient for you.

Are there such differences for Croatia?

**Reply by mnalis** [reply]
> Well in case of Germany the differences result in you're legally able to ride your bike over them, or you have to get off it and walk across, which makes a difference in travel time for routing - and obviously convenience ;)

Yes, same in Croatia. For example, in your [example bottom picture](https://github.com/streetcomplete/StreetComplete/discussions/5875#discussioncomment-10559375); bicycle can be ridden across the road using it (over that red-colored way with bicycle pictogram).
However, **if** that red-colored cycleway on right side **was missing** (i.e. there was only black-and-white zebra on the left) the **cyclist would have to dismount** and cross the crossing as pedestrian, pushing the bike on the side instead of riding on it.

However, if cyclist had to dismount due to lack of that red-colored cycleway, in Croatia's OSM it would be tagged as [bicycle=dismount](https://wiki.openstreetmap.org/wiki/Tag:bicycle=dismount)  (instead of `bicycle=designated` + `segregated=yes` which would be used is bicycles are allowed) on that way which crosses the road. E.g. [way 692406106](https://www.openstreetmap.org/way/692406106).

Thus, any microtagging of  `crossing:markings` / `footway:crossing:markings` / `cycleway:crossing:markings` in Croatia would be completely optional and **without any effect on routing**.  (as I guess they would be elsewhere -- those tags can only provide more detailed reason _why_ cyclist had to dismount; but the dismounting itself is due to `bicycle=dismount` tag. And as in Croatia only reason is _"because there is no cycleway marked, duh!"_, microtagging it explicitly it would not bring much extra detail to the table.)

> Same for walking, we got zebra crossings, where you have priority as pedestrian, or regular crossings, where you may have to wait for a minute or so to cross.

We don't have different pedestrian crossings, no. Huge majority (if not all) of all pedestrian crossings are marked with zebra stripes[^1],  and that does not have any effect on priority. I.e. how long you have to wait depends how municipality set up traffic light timers (and they have free reign there - which is heavily in favor of motorcars, unfortunately) or on luck & local traffic conditions (if no traffic lights are present). 

Only other alternative are non-marked places; and as long as place where pedestrian wants to cross the road is 50 or 100m away from existing zebra crossing, pedestrians can legally cross the road anywhere (_"after making sure that can do it in a safe way"_) [^2]. That might be faster or slower than waiting on zebra with traffic lights, depending on your luck and traffic conditions.

[^1]: Ref. [_ZOSPNC Članak 130._](https://www.zakon.hr/z/78/Zakon-o-sigurnosti-prometa-na-cestama)
[^2]: Ref. _Članak 129._
